src/kaleidoscope/KeyAddrBitfield.h

```diff
@@ -0,0 +1,184 @@
+/* Kaleidoscope - Firmware for computer input devices
+ * Copyright (C) 2020  Keyboard.io, Inc.
```

Comment:
Should this copyright date include only the first date for this particular file, as I've done it here? I see some other files which are not as old as their copyright dates, which did not seem appropriate to me.

Reply:
When adding copyright years, I usually go with when the _code_ in the file was first introduced. If it was in some other file first, use that for the copyright year. But the current year isn't wrong either.

Reply:
Hmm. I did lift this code from something I wrote a couple of years ago, but  then I wonder if the assignment of the copyright to Keyboardio is appropriate. Maybe I should add a copyright line above this one, with my name on it? I'm fine with it either way, really; I just like to get things "right".